### ISSUE TITLE
Cannot open tag and date editor or focus task with keyboard when creating task

### ISSUE THREAD

**Issue by mt-xing** [cornell-dti/samwise#435]
**Describe the bug**
When creating a task, tab over to any of the three editors (focus, tag, or date). Neither space or keyboard actually activates them, thus rendering the tabbing useless.

**To Reproduce**
Try to create a new task
Press tab a few times to highlight one of the three editors
Press space or enter
Nothing happens
